src/service/action-impl.js

```diff
@@ -121,6 +126,39 @@ export class ActionService {
   }
 
   /**
+   * Installs action handler for the specified element.
+   * @param {!Element} target
+   * @param {function(!ActionInvocation)} handler
+   */
+  installActionHandler(target, handler) {
+    const debugid = target.tagName + '#' + target.id;
+    assert(this.isAmpElement_(target), 'AMP element is expected: %s', debugid);
```

Comment:
Not in this case. This is a non-authoring visible error. Unfortunately, `assert` and associate elements are a bit abused. I believe we are tracking it somewhere in one of the issues. Right, @erwinmombay ?